Source/UnitTests/GTMSessionFetcherURLSessionDelegateSwapTest.m

```diff
@@ -0,0 +1,117 @@
+//
```

Comment:
Please use the standard header for sources, we can't take a patch with a specific copyright that doesn't follow the project licensing.

Reply:
thanks